project/Projects.scala

```diff
@@ -20,5 +20,5 @@ object Projects {
   lazy val javasrc2cpg = project.in(frontendsRoot / "javasrc2cpg")
   lazy val jimple2cpg  = project.in(frontendsRoot / "jimple2cpg")
   lazy val kotlin2cpg  = project.in(frontendsRoot / "kotlin2cpg")
-  lazy val rubysrc2cpg = project.in(frontendsRoot / "rubysrc2cpg")
+  lazy val rubysrc2cpg    = project.in(frontendsRoot / "rubysrc2cpg")
```

Comment:
Unless this was scalafmt's doing:

```suggestion
  lazy val rubysrc2cpg = project.in(frontendsRoot / "rubysrc2cpg")
```

Reply:
Somehow `scalafmt` wasn't formatting this. Did this manually and `scalafmt` did not change it